plugin/settings/checkbox.php

```diff
@@ -13,28 +13,52 @@
  * This will become unnecessary when using the new AJAX/API/Form module, because
  * they use a smart JavaScript function to gather form fields data.
  */
-function render_setting_field_checkbox($config) {
+function render_setting_field_checkbox($config, $type='checkbox') {
```

Comment:
Could add a comment in the docblock above, what values are accepted for new this function argument?

Reply:
Actually, I think I'd prefer this to be a `type` property of the `$config` argument.